lib/chef/event_dispatch/dispatcher.rb

```diff
@@ -20,6 +20,14 @@ def register(subscriber)
         @subscribers << subscriber
       end
 
+      # Check to see if we are dispatching to a formatter
+      def formatter?
+        @subscribers.each do |s|
```

Comment:
This has odd semantics with multiple subscribers. Wouldn't we want to check if either `any?` or `all?` rather than just the first that happens to respond?